OpenRA.Mods.Common/Traits/Render/RenderDetectionCircle.cs

```diff
@@ -29,8 +29,14 @@ class RenderDetectionCircleInfo : TraitInfo, Requires<DetectCloakedInfo>
 		[Desc("Color of the circle and scanner update line.")]
 		public readonly Color Color = Color.FromArgb(128, Color.LimeGreen);
 
+		[Desc("Range circle line width")]
```

Comment:
The newly added descs don't match the wording of these older ones. Can we please either reword either the old or new ones so they are a consistent style? Also note my previous comments about outline and contrast.